src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/histogram.js

```diff
@@ -254,11 +253,12 @@ class Histogram extends Visualization {
                 [0, 0],
                 [canvas.inner.width, canvas.inner.height],
             ])
-            .on(zoomClass, zoomed)
+            .on('zoom', zoomed)
+        // .on('mousewheel', zoomed)
```

Comment:
Commented code.

Reply:
I have no idea what it does here haha, i thought i've removed it ...